dashboard/db/schema.rb

```diff
@@ -1409,7 +1410,7 @@
     t.index ["user_id"], name: "index_queued_account_purges_on_user_id", unique: true
   end
 
-  create_table "reference_guides", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci", force: :cascade do |t|
+  create_table "reference_guides", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci", force: :cascade do |t|
```

Comment:
I know what happened here ... fixing it now

Reply:
:joysob:

Reply:
Lol ... I actually think this one wasn't totally my fault 😅